src/main/java/Duke/Task/Events.java

```diff
@@ -0,0 +1,108 @@
+/**
+ * @author Hang Zelin
+ * <p>
+ * Events class that extends Task class. It is one of the types in 3 tasks.
+ * Will contain a time in the form of "/by"
+ */
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
+
+public class Events extends Task {
+
+    private boolean done;
+    private String task;
+    private LocalDateTime time;
+    private String taskType = "E";
```

Comment:
yup so like the above, i think alot of variables can be FINAL. Id stop mentioning it